starter-core/src/main/java/io/micronaut/starter/feature/function/gcp/template/gcpFunctionReadme.rocker.raw

```diff
@@ -35,19 +37,23 @@ $ cd @features.build().getJarDirectory()
 Now run:
 
 ```bash
-$ gcloud alpha functions deploy @project.getName() --entry-point io.micronaut.gcp.function.http.HttpFunction --runtime java11 --trigger-http
+@if(isFunction){
+$ gcloud functions deploy @project.getName() --entry-point @if(project.getPackageName() != null){@(project.getPackageName()).}Function --runtime java11 --trigger-http
+} else {
+$ gcloud functions deploy @project.getName() --entry-point io.micronaut.gcp.function.http.HttpFunction --runtime java11 --trigger-http
+}
 ```
 
 Choose unauthenticated access if you don't need auth.
 
 To obtain the trigger URL do the following:
 
 ```bash
-$ YOUR_HTTP_TRIGGER_URL=$(gcloud alpha functions describe @project.getName() --format='value(httpsTrigger.url)')
+$ YOUR_HTTP_TRIGGER_URL=$(gcloud functions describe @project.getName() --format='value(httpsTrigger.url)')
 ```
 
 You can then use this variable to test the function invocation:
 
 ```bash
-$ curl -i $YOUR_HTTP_TRIGGER_URL/hello
+$ curl -i $YOUR_HTTP_TRIGGER_URL/
```

Comment:
Instead of deleting `/hello` we should probably replace it with `/@project.getPropertyName()`

Reply:
I've added `/@project.getPropertyName()`